opentelemetry-api/src/opentelemetry/_metrics/__init__.py

```diff
@@ -423,7 +428,9 @@ class NoOpMeter(Meter):
     def create_counter(self, name, unit="", description="") -> Counter:
         """Returns a no-op Counter."""
         super().create_counter(name, unit=unit, description=description)
-        if self._check_instrument_id(name, DefaultCounter, unit, description):
+        if self._is_instrument_registered(
```

Comment:
Does this check work if `self._is_instrument_registered` returns a `tuple` now?

Reply:
Right, fixed